addons/mail/static/src/core/common/im_status.js

```diff
@@ -2,7 +2,7 @@ import { Component } from "@odoo/owl";
 import { Typing } from "@mail/discuss/typing/common/typing";
 
 export class ImStatus extends Component {
-    static props = ["persona", "className?", "style?", "thread?"];
+    static props = ["persona", "className?", "style?", "member?"];
```

Comment:
I would make `persona` optional when `member` is provided, because the persona of member is trivially accessible.

This would simplify using `<ImStatus/>` with member to just pass member rather than also pass `persona`.

Reply:
The edge case for Partner Suggestions. There we have only Persona.

Reply:
Ok to keep `thread?` just for that one. The rest can use `member` without providing a `persona` as prop

Reply:
I don't think providing`thread` instead of `persona` would help. Because you can't determine for which partner this ImStatus is to be displayed.

Reply:
No, I meant for this specific case of Partner suggestion, it's ok to pass both `persona` and `thread?`